.pre-commit-config.yaml

```diff
@@ -4,7 +4,7 @@ repos:
   rev: 18.9b0
   hooks:
   - id: black
-    language_version: python3.6
+    language_version: python3
```

Comment:
This specific lines annoys me slightly, because it will fail with a less nice error for people on py35. 

I must check if pre-commit supports specifying a minimum language version.

Reply:
I use python 3.7 (Fedora here) and without this patch I cannot commit. If you have a better approach, it's OK. :roll_eyes: